src/Database/Adapter/MariaDB.php

```diff
@@ -141,6 +141,43 @@ public function createCollection(string $name, array $attributes = [], array $in
         return true;
     }
 
+    /**
+     * Get Collection Size
+     * @param string $name
+     * @return int
+     * @throws Exception
+     * 
+     */
+
+     public function getCollectionSize(string $name): int
```

Comment:
Maybe we need to a datatype bigger than int?

Reply:
I changed the return type to string.